modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -41,82 +37,61 @@ EOD
+servicebinding.binding.operators.coreos.com/spring-petclinic created
 ----
 
-. To verify that the binding is successful, check the status conditions of the binding resource:
+. Verify that the request for service binding is successful:
 +
+[source,terminal]
 ----
-$ oc get servicebindings spring-petclinic-rest -n my-postgresql -o jsonpath-as-json='{.status.conditions}'
+$ oc get servicebindings -n my-petclinic
 ----
 +
 .Example output
+[source,terminal]
 ----
-[
-  [
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "DataCollected",
-      "status": "True",
-      "type": "CollectionReady",
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationUpdated",
-      "status": "True",
-      "type": "InjectionReady",
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationsBound",
-      "status": "True",
-      "type": "Ready",
-    },
-  ],
-]
+NAME                          READY   REASON              AGE
+spring-petclinic-postgresql   True    ApplicationsBound   47m
 ----
 +
-By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application.
-
-. To verify that the files in the application contain the projected binding data, use the following command in shell to print out the contents:
+By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application. For example, all the values from the Secret resource are projected into the `bindings/spring-petclinic-pgcluster` directory.
 +
+[NOTE]
+====
+Optionally, you can also verify that the files in the application contain the projected binding data, by printing out the directory contents:
+
+[source,terminal]
 ----
-$ for i in username password host port type; do oc exec -it deploy/spring-petclinic-rest -n my-postgresql -- /bin/bash -c 'find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
+$ for i in username password host port type; do oc exec -it deploy/spring-petclinic -n my-petclinic -- /bin/bash -c 'cd /tmp; find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
 ----
-+
+
 .Example output: With all the values from the secret resource
+[source,text]
 ----
-/bindings/spring-petclinic-rest/username: hippo
-/bindings/spring-petclinic-rest/password: w0ZB<0j1W|K;+4*TlK7-w^z/
-/bindings/spring-petclinic-rest/host: hippo-pgbouncer
-/bindings/spring-petclinic-rest/port: 5432
-/bindings/spring-petclinic-rest/type: postgresql
+/bindings/spring-petclinic/username: hippo
+/bindings/spring-petclinic/password: w0ZB<0j1W|K;+4*TlK7-w^z/
+/bindings/spring-petclinic/host: hippo-primary.my-petclinic.svc
+/bindings/spring-petclinic/port: 5432
+/bindings/spring-petclinic/type: postgresql
```

Comment:
```suggestion
/bindings/spring-petclinic-pgcluster/username: hippo
/bindings/spring-petclinic-pgcluster/password: w0ZB<0j1W|K;+4*TlK7-w^z/
/bindings/spring-petclinic-pgcluster/host: hippo-primary.my-petclinic.svc
/bindings/spring-petclinic-pgcluster/port: 5432
/bindings/spring-petclinic-pgcluster/type: postgresql
```
The name of the binding directory should match the name of the `ServiceBinding` resource

Reply:
@Srivaralakshmi ^^

Reply:
Done.